src/enums/MatchStatus.ts

```diff
@@ -0,0 +1,6 @@
+export enum MatchStatus {
+  live = 'LIVE',
```

Comment:
Enums in TypeScript are capitalized by convention.

Reply:
Done.